src/gui_impl/main_window.cpp

```diff
@@ -44,7 +44,7 @@ MainWindow::MainWindow(QSharedPointer<ApplicationContext> application_context)
     setUnifiedTitleAndToolBarOnMac (true);
     
     QSettings settings("SigViewer");
-    resize(settings.value("MainWindow/size", QSize(900, 800)).toSize());
+    resize(settings.value("MainWindow/size", QSize(1500, 800)).toSize());
```

Comment:
Also, the current default color for non-XDF file is quite dark. Should we consider using a more vivid blue or green color, what do you think Clemens?

![image](https://cloud.githubusercontent.com/assets/20526646/23643261/9a0258d8-02ce-11e7-908d-b60fa5145353.png)